.github/workflows/upload.yml

```diff
@@ -204,6 +204,45 @@ jobs:
         run: |
           python -m pytest tests -m qchem -n auto
 
+  data-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.4.1
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip && pip install wheel --upgrade
+          pip install -r requirements-ci.txt --upgrade
+          pip install -r requirements-dev.txt --upgrade
+
+      - name: Install zstd
+        run: pip3 install zstd
+
+      - name: Install dill
+        run: pip3 install dill
```

Comment:
Does @josh146 mean we could add these lines to the old tests?

Reply:
Hey @DSGuala, I think that is the idea. 
You could install the dependencies in the workflows that include the tests that will need them.